contracts/interfaces/accounts/ISafe.sol

```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity ^0.8.18;
```

Comment:
We potentially could do the same here as we did for components (i.e. use a file instead of a folder)